admin_guide/backup_restore.adoc

```diff
@@ -200,6 +200,77 @@ that {product-title} was previously installed.
 $ oc create -f mycluster.template.yaml
 ----
 
+[[backup-restore-recovering-etcd-hosts]]
+== Recovering etcd Hosts
+
+In cases where etcd hosts have failed, but you have at least one host still running, you can use that one host to recover etcd hosts without downtime.
+
+[NOTE]
+====
+The following presumes you have backups of the */etc/etcd* configuration for the etcd hosts.
+====
+
+. Add the desired number of hosts to the cluster.
+
+. Upgrade etcd in the surviving node with the provided version:
++
+----
+# rpm -Fvh etcd-2.3.7-3.el7.bz1350875.1.x86_64.rpm
+----
+
+. Create the new single node cluster using etcd’s --force-new-cluster option. You can do this with a long complex command using the values from /etc/etcd/etcd.conf, or you can temporarily modify the systemd unit file and start the service normally.
++
+To do so, edit the /usr/lib/systemd/system/etcd.service and add --force-new-cluster:
++
+----
+# sed -i '/ExecStart/s/"$/  --force-new-cluster"/' /usr/lib/systemd/system/etcd.service
+# cat /usr/lib/systemd/system/etcd.service  | grep ExecStart
+
+ExecStart=/bin/bash -c "GOMAXPROCS=$(nproc) /usr/bin/etcd --force-new-cluster"
+----
+
+. Remove `--force-new-cluster` from the systemd unit file.
+
+. Install the etcd rpm on the new hosts:
++
+----
+# rpm -ivh etcd-2.3.7-3.el7.bz1350875.1.x86_64.rpm
+----
+
+. Restore */etc/etcd* from the backup onto the new hosts.
+
+. Add one of the hosts to the cluster:
++
+----
+# etcdctl member add <hostname> <IP_address>
+----
++
+This will give you an ID for the new host.
+
+. On the node host that did not fail, create a backup of the database with the two new options that this fix adds:
++
+----
+# etcdctl backup --keep-cluster-id --node-id NODEID --backup-dir /path/to/backup/dir --data-dir /var/lib/etcd
+----
+
+. Transfer */path/to/backup/dir* to node 2 and use it as its etcd data dir (usually */var/lib/etcd*).
+
+. Set the correct rights for the etcd files:
++
+----
+chown -R etcd:etcd /var/lib/etcd/
+----
+
+. Start etcd on the new hosts.
+
+. To verify that the new etcd host is healthy:
++
+----
+# etcdctl cluster-health
+----
+
+. Repeat steps 7-11 to add more etcd hosts.
```

Comment:
I don't see were we are labeling / numbering the steps above. Do we need to update this to make it more clear?